docs/_sections/forms.html

```diff
@@ -87,6 +87,34 @@ <h3 id="requiredFields">Required fields</h3>
 &lt;/div&gt;
     </code></pre>
 
+    <h3 id="fieldErrors">Error state on fields</h3>
+
+    <p>Any field where user input could be deemed or returned erroneous should utilize the class <code>nypl-field-error</code>. It is up to the form designer to determine what fields should have an error state. It is up to the developer to implement the necessary front and back end validations as well as the expected input formatting that would necessitate a state change for a given field. Additionally, explanatory text must be supplied by the form designer decribing the error and how to correct it. </p>
+
```

Comment:
also specify that a `span.error-desc` should be appended to the main field element via JavaScript (or other way) with the necessary explanation text (if the field has no error, it just reflects a normal text field but once there is error a new `span` must appear, either via a page refresh or JS)